dubbo-remoting/dubbo-remoting-netty4/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyClient.java

```diff
@@ -96,11 +96,7 @@ protected void doOpen() throws Throwable {
                 //.option(ChannelOption.CONNECT_TIMEOUT_MILLIS, getTimeout())
                 .channel(NioSocketChannel.class);
 
-        if (getConnectTimeout() < 3000) {
-            bootstrap.option(ChannelOption.CONNECT_TIMEOUT_MILLIS, 3000);
-        } else {
-            bootstrap.option(ChannelOption.CONNECT_TIMEOUT_MILLIS, getConnectTimeout());
-        }
+        bootstrap.option(ChannelOption.CONNECT_TIMEOUT_MILLIS, Math.max(3000,getConnectTimeout()));
```

Comment:
It's better to keep code style same as other part.
I suggest add a space before `getConnectTimeout()`

Reply:
ok